portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,15 +105,17 @@ private String convertToJsonUsingGson(ArrayList<Comment> comments, int numCommen
     return json;
   }
 
+  // creates a comment object from the form data
   private Comment makeComment(HttpServletRequest request) {
-      String name = request.getParameter("name");
+      String name = userService.getCurrentUser().getUserId();
       String comment = request.getParameter("comment");
       if (name == null && comment == null)
         return null;
       else
-        return new Comment(name, comment, userService.getCurrentUser().getEmail());
+        return new Comment(name, comment);
```

Comment:
Should timestamp be a field?

Reply:
in this case no, the date is added in the comment constructor